python/tank/errors.py

```diff
@@ -59,8 +59,3 @@ def __init__(self):
         """
         super(TankErrorProjectIsSetup, self).__init__("You are trying to set up a project which has already been set up. "
                                                       "If you want to do this, make sure to set the force parameter.")
-
-# backwards compatibility to ensure code that was calling internal
-# parts of the API will still work.
```

Comment:
Hmm, this is supposed to be a backwards compatibility thing. Will this break things? 

Please double check that the following still works:

```
>>> import tank
>>> tank.TankEngineInitError
<class 'tank.platform.errors.TankEngineInitError'>
>>> tank.errors.TankEngineInitError
<class 'tank.platform.errors.TankEngineInitError'>
>>> tank.platform.TankEngineInitError
<class 'tank.platform.errors.TankEngineInitError'>
>>> tank.platform.errors.TankEngineInitError
<class 'tank.platform.errors.TankEngineInitError'>
```